rust/src/ftp/ftp.rs

```diff
@@ -161,7 +160,7 @@ impl Default for FtpTransferCmd {
             file_name: std::ptr::null_mut(),
             file_len: 0,
             direction: 0,
-            cmd: FtpStateValues::FTP_STATE_NONE as u8,
```

Comment:
why is this changed? It seems a named value is clearer

Reply:
It's an incorrect value -- it's expected to be a command enum ... not a state value

Reply:
should it be FTP_COMMAND_UNKNOWN then?

Reply:
Maybe? It's always initialized before use -- the current code calloc's the memory for this struct which is why I used `0`

Reply:
ok lets leave as-is